app/controllers/accounts_controller.rb

```diff
@@ -26,7 +26,7 @@ def edit
   # POST /accounts.json
   def create
     respond_to do |format|
-      if @account.save_and_create_tenant
+      if CreateAccount.new(@account).save
```

Comment:
I have a bias towards noun-y service class names, i.e. `AccountCreator` vs. `CreateAccount`, but meh, I wouldn't hold up this PR over that.